Assets/MRTK/Core/Utilities/Editor/SpatializerUtilities.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Editor;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Editor
+{
+    /// <summary>
+    /// Collection of utilities to manage the configured audio spatializer.
+    /// </summary>
+    public static class SpatializerUtilities
+    {
+        /// <summary>
+        /// Returns the name of the currently selected spatializer plugin.
+        /// </summary>
+        public static string CurrentSpatializer => AudioSettings.GetSpatializerPluginName();
+
+        /// <summary>
+        /// Returns the names of installed spatializer plugins.
+        /// </summary>
+        public static string[] InstalledSpatializers => AudioSettings.GetSpatializerPluginNames();
+
+        /// <summary>
+        /// Checks to see if the audio spatializer is configured and/or whether or
+        /// not the spatializer collection has changed.
+        /// </summary>
+        /// <returns>
+        /// True if; the selected spatializer is installed and no changes have been made to the collection of installed spatializers.
+        /// False if; There is no selected spatializer, the selected spatializer is no longer installed or the collection of installed spatializers has been changed.
+        /// </returns>
+        public static bool CheckSettings()
+        {
+            string spatializerName = CurrentSpatializer;
+
+            // Check to see if an audio spatializer is configured.
+            if (string.IsNullOrWhiteSpace(spatializerName)) 
+            {
+                // A spatializer has not been configured.
+                return false; 
+            }
+
+            string[] installedSpatializers = AudioSettings.GetSpatializerPluginNames();
+
+            // Check to see if the configured spatializer is installed.
+            if (!installedSpatializers.Contains<string>(spatializerName))
+            {
+                // The current spatializer has been uninstalled.
+                return false;
+            }
+
+            // Next, check to see if the count of installed spatializers has changed
+            if (!CheckSpatializerCount())
```

Comment:
So I guess this utility wont catch changes to the list where the number stays the same, but the plugins change right?

Reply:
Correct. That was out of the scope of this implementation. It is likely a fairly corner case (ex: uninstall unused spatializer x and install new spatializer y) between project loads.